### PR TITLE
showing txn in pending state

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- email gets sent to user on clicking payout (doesn't wait for confirmation)
- entry in transaction tab is marked as pending state 

image

### CODE REVIEW

**Comment by thelostone-mc** [comment]
@owocki quick check -> the amount should get updated as well! I'm not sure what that happens

File: app/grants/sync/helpers.py [hunk 1]
```diff
@@ -55,7 +55,8 @@ def record_contribution_activity(contribution):
                 comment= subscription.comments
             )
 
-        successful_contribution(grant, subscription, contribution)
+        # note: commenting out for optimistic UI
+        # successful_contribution(grant, subscription, contribution)
```
Comment (owocki) [inline app/grants/sync/helpers.py:59]: where is this moved to?  seems like it's not re-added anywhere

File: app/grants/views.py [hunk 2]
```diff
@@ -2698,7 +2698,9 @@ def contribute_to_grants_v1(request):
             subscription.save()
 
             # step 3: create contribution + fire celery
-            contribution = subscription.create_contribution(tx_id, False)
+            contribution = subscription.create_contribution(tx_id, True)
+            contribution.success = True
```
Comment (owocki) [inline app/grants/views.py:2702]: is it marked as failed if the contribution fails elsewhere?

**Comment by owocki** [comment]
which of the feedback items in my ticket ( https://github.com/gitcoinco/web/issues/7796 ) does this fix?  just the tx history tab, or the other items too?

**Comment by owocki** [comment]
which of the feedback items in my ticket ( https://github.com/gitcoinco/web/issues/7796 ) does this fix?  just the tx history tab, or the other items too?

File: app/grants/views.py (hunk 2 above)
Comment (thelostone-mc) [reply app/grants/views.py:2702]: nope! there is no way for us to fail -> hence it'll be in forever pending 
The fallback we could have is to auto fail it after 5 hours

File: app/grants/sync/helpers.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/sync/helpers.py:59]: this is taken care  https://github.com/gitcoinco/web/pull/7797/files#diff-f630b64d112cbde98b1992af074c13f88ce29c0f6a81dffdc2fdfa967fb8a2c6R2701

changing it to true adds it

`contribution = subscription.create_contribution(tx_id, True)`

**Review by danlipert — APPROVED** [review]

**Pull request merged**